src/deployments.ts

```diff
@@ -23,7 +23,7 @@ import {
 } from './types';
 
 export const DEPLOYMENT_AZRAEL_ETHEREUM_MAINNET_V0 = {
-  contractAddress: '0x94d8f036a0fbc216bb532d33bdf6564157af0cd7',
+  contractAddress: '0x94D8f036a0fbC216Bb532D33bDF6564157Af0cD7',
```

Comment:
Looks like we've spotted more addresses which weren't checksummed. Have you checked to see whether this change will invalidate the frontend?